roles/hxr.admin-tools/tasks/main.yml

```diff
@@ -1,24 +1,12 @@
 ---
 - name: Install useful tools
   yum:
-    name: "{{ admin_packages }}"
+    name: "{{ admin_packages | select | list }}"
     state: installed
 
-- name: Check if firewalld is already installed
-  yum:
-    list: firewalld
-  # If not installed yum_list.results[*].yumstate != installed
-  register: yum_list
-
-- name: Stop firewalld
-  service:
-    name: firewalld
-    state: stopped
-    enabled: no
-  when: yum_list.results | selectattr("yumstate", "match", "installed") | list | length != 0
```

Comment:
> This Role is used by:
> - telescope
> - apollo
> - build
> - plausible
> - cvmfs

Make sure that none of them assume that firewalld is disabled.

Reply:
I think we do not want to disable firewalld anywhere.
since this is read only friday, I will not touch other roles today. We can let this unmerged for now

Reply:
Yes I also think we do not want to disable firewalld anywhere. I am just commenting it so that we remember this.